doc/ref_model/chapters/chapter05.md

```diff
@@ -19,7 +19,7 @@
   * [5.4.3 Network Resources.](#5.4.3)
 
 
-A profile [RM Section 2.4](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#24-profiles--flavours) specifies the configuration of a cloud infrastructure node (host or server); [profile extensions](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#242-profile-extensions-specialisations) may specify additional configuration. Workloads utilise profiles to describe the configuration of nodes on which they can be hosted to execute on. Workload Flavours provide a mechanism to specify the VM or Pod sizing information to host the workload.  Depending on the requirements of the workloads, a VM or a Pod will be deployed as per the specified Flavour information on a node configured as per the specified Profile. Not only do the nodes (the hardware) have to be configured but some of the capabilities also need to be configured in the software layers (such as Operating System and Virtualisation Software). Thus, a Profile can be defined in terms of configuration needed in the software layers, the Cloud Infrastructure Software Profile, and the hardware, the Cloud Infrastructure Hardware Profile.
+A profile [RM Section 2.4](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#24-profiles--flavours) specifies the configuration of a Cloud Infrastructure node (host or server); [profile extensions](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#242-profile-extensions-specialisations) may specify additional configuration. Workloads utilise profiles to describe the configuration of nodes on which they can be hosted to execute on. Workload Flavours provide a mechanism to specify the VM or Pod sizing information to host the workload.  Depending on the requirements of the workloads, a VM or a Pod will be deployed as per the specified Flavour information on a node configured as per the specified Profile. Not only do the nodes (the hardware) have to be configured but some of the capabilities also need to be configured in the software layers (such as Operating System and Virtualisation Software). Thus, a Profile can be defined in terms of configuration needed in the software layers, the Cloud Infrastructure Software Profile, and the hardware, the Cloud Infrastructure Hardware Profile.
```

Comment:
```suggestion
A profile [RM Section 2.4](chapter02.md#24-profiles--flavours) specifies the configuration of a Cloud Infrastructure node (host or server); [profile extensions](chapter02.md#242-profile-extensions-specialisations) may specify additional configuration. Workloads utilise profiles to describe the configuration of nodes on which they can be hosted to execute on. Workload Flavours provide a mechanism to specify the VM or Pod sizing information to host the workload.  Depending on the requirements of the workloads, a VM or a Pod will be deployed as per the specified Flavour information on a node configured as per the specified Profile. Not only do the nodes (the hardware) have to be configured but some of the capabilities also need to be configured in the software layers (such as Operating System and Virtualisation Software). Thus, a Profile can be defined in terms of configuration needed in the software layers, the Cloud Infrastructure Software Profile, and the hardware, the Cloud Infrastructure Hardware Profile.
```
Please always use relative links. Absolute links contain the branch of the document, so they will be incorrect after the release is moved to a new branch.